### PR TITLE
Updates to dbt Explorer

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Updates to dbt Explorer
- Update this page https://docs.getdbt.com/docs/collaborate/explore-projects with the latest changes
- Create a new subpage and move this section https://docs.getdbt.com/docs/collaborate/explore-projects#about-project-level-lineage and all its subsections to it
- Create a release note about this update

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Needs review from PM

Adding new pages (delete if not applicable):
- ~~[ ] Add page to `website/sidebars.js`~~ n/a for release notes
- [x] Provide a unique filename for the new page

### CODE REVIEW

**Review by dave-connors-3 — APPROVED** [review]
🦞 LGTM with those two removals we chatted about!

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+tags: [Nov-2023]
+---
+
+dbt Labs is excited to announce the latest features and UI updates to dbt Explorer!
+
+For more details, refer to [Explore your dbt projects](/docs/collaborate/explore-projects).
+
+## The project's lineage graph 
+
+- The search bar in the full lineage graph is now to the right of the graph and is larger.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - The search bar in the full lineage graph is now more prominent.

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+dbt Labs is excited to announce the latest features and UI updates to dbt Explorer!
+
+For more details, refer to [Explore your dbt projects](/docs/collaborate/explore-projects).
+
+## The project's lineage graph 
+
+- The search bar in the full lineage graph is now to the right of the graph and is larger. 
+- The project/account navigation breadcrumbs has been updated.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - It's easier to navigate across projects using the breadcrumbs.

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- A context menu is now available so you can explore a node's lineage and its details more easily from the full lineage graph.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - The new context menu (right click) makes it easier to focus on a node or view its lineage.

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<LoomVideo id='1d28cc6443a94f00bbcb43d6ab3ad0d9' />
+
+## Search improvements 
+
+- When searching with keywords, a new side panel UI to filter search results by resource type.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - When searching with keywords, a new side panel UI helps filter search results by resource type, tag, column, and other key properties (in lieu of manually defining selectors).

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- The search bar in the full lineage graph is now to the right of the graph and is larger. 
+- The project/account navigation breadcrumbs has been updated.
+- A context menu is now available so you can explore a node's lineage and its details more easily from the full lineage graph.
+
+<LoomVideo id='1d28cc6443a94f00bbcb43d6ab3ad0d9' />
+
+## Search improvements 
+
+- When searching with keywords, a new side panel UI to filter search results by resource type. 
+- Improved search for only exact column names.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - Search result logic is clearly explained, for instance whether a resource contains a column name (exact match only)

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+<LoomVideo id='1d28cc6443a94f00bbcb43d6ab3ad0d9' />
+
+## Search improvements 
+
+- When searching with keywords, a new side panel UI to filter search results by resource type. 
+- Improved search for only exact column names. 
+
+<LoomVideo id='5c4dc30afa824897a8c8b3ed37cee782?t=14' />
+
+## Browse with sidebar
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: Please remove this section. No notable updates.

Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: Please remove this section. No notable updates.

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- There are UI updates to browsing your resources with the sidebar.
+
+<LoomVideo id='eb7aad34b581483fb340ab7ef9aac2e0' />
+
+## Resource details
+- In the resource details for models, there are UI updates to the **Tests** section and the **Columns** tab.
```
Comment (cafzal) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: - Model test result statuses are displayed on the model details page. 
- Column names can now be searched within the list.

**Review by cafzal — CHANGES_REQUESTED** [review]
Minor edits to release notes

**Review by cafzal — APPROVED** [review]
looks good, thanks!

File: website/docs/docs/collaborate/explore-projects.md [hunk 8]
```diff
@@ -53,17 +53,23 @@ To interact with the full lineage graph, you can:
...
+    - Refocus on the node and its children only
+    - Refocus on the node and it parents only
+    - View the node's [resource details](#view-resource-details) page
+
 - Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
     - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
     - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
-- [Search and select specific resources](#search-resources) or a subset of the DAG using selectors and graph operators. For example:
+- [Search and select specific resources](#search-resources) or a subset of the DAG using [selectors](/reference/node-selection/methods) and [graph operators](/reference/node-selection/graph-operators). This can help you narrow the focus on the resources that interest you. For example:
     - `+[RESOURCE_NAME]` &mdash; Displays all parent nodes of the resource
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-projects.md:66]: non blocking sugg - i like the example and i would suggest sharing a real-life example that users new to explorer will immediately understand, as opposed to only displaying a generic 'resource_type:model [resource_name]` example.

Comment (mirnawong1) [reply website/docs/docs/collaborate/explore-projects.md:66]: similar to how you display an example in 'Search with selectors'

File: website/docs/docs/collaborate/explore-projects.md [hunk 9]
```diff
@@ -74,9 +80,13 @@ Select a node (single-click) in the lineage graph to highlight its relationship
 ### Search with keywords
 When searching with keywords, dbt Explorer searches through your resource metadata (such as resource type, resource name, column name, source name, tags, schema, database, version, alias/identifier, and package name) and returns any matches.
 
-### Search with selector methods 
+With keyword search, there is a side panel (to the right of the main section) that you can use to additionally filter the search results by resource type. You can filter by resource tags for all resources or model access levels within the **Models** option. As an example, when searching for “sale”, the results will include all resources with the keyword “sale” in their metadata. And, choosing **Models** and **Sources** in the side panel will filter those search results to contain only models or sources.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-projects.md]: sugg bullets or shortening so it's easier to read/scan

```suggestion
- Keyword search features a side panel (to the right of the main section) to filter search results by resource type. 
- Use this panel to select specific resource tags or model access levels under the **Models** option. 
  - For example, a search for "sale" returns results that include all resources with the keyword "sale" in their metadata. Filtering by **Models** and **Sources** refines these results to only include models or sources.
```

File: website/docs/docs/collaborate/explore-projects.md [hunk 10]
```diff
@@ -74,9 +80,13 @@ Select a node (single-click) in the lineage graph to highlight its relationship
 ### Search with keywords
 When searching with keywords, dbt Explorer searches through your resource metadata (such as resource type, resource name, column name, source name, tags, schema, database, version, alias/identifier, and package name) and returns any matches.
 
-### Search with selector methods 
+With keyword search, there is a side panel (to the right of the main section) that you can use to additionally filter the search results by resource type. You can filter by resource tags for all resources or model access levels within the **Models** option. As an example, when searching for “sale”, the results will include all resources with the keyword “sale” in their metadata. And, choosing **Models** and **Sources** in the side panel will filter those search results to contain only models or sources.   
+
+For a search on an exact column name, the results will include all relational nodes with that specific column in their schemas. If there is a match, the search result will include a notice indicating that the resource contains the specified column.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-projects.md]:
```suggestion
- When searching for an exact column name, the results show all relational nodes containing that column in their schemas. If there's a match, a notice in the search result indicates the resource contains the specified column.
```

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Enhancement: New features and UI changes to dbt Explorer"
+description: "November 2023: New features and UI changes to dbt Explorer"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md]: sugg adding more description so users have an insight into the quick changes for curiosity
```suggestion
description: "November 2023: New features and UI changes to dbt Explorer, including a new filter panel, improved lineage graph, and detailed resource information."
```

File: website/docs/docs/collaborate/explore-multiple-projects.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Explore multiple projects"
+sidebar_label: "Explore multiple projects"
+description: "Learn about project-level lineage in dbt Explorer and its uses."
+pagination_next: null
+---
+
+You can also view all the different projects and public models in the account, where the public models are defined, and how they are used to gain a better understanding about your cross-project resources.
+
+When viewing the resource-level lineage graph for a given project that uses cross-project references, you can see cross-project relationships represented in the DAG. The iconography is slightly different depending on whether you're viewing the lineage of an upstream producer project or a downstream consumer project.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-multiple-projects.md]:
```suggestion
The resource-level lineage graph for a given project displays the cross-project relationships in the DAG. The different icons indicate whether you’re looking at an upstream producer project (parent) or a downstream consumer project (child).
```

File: website/docs/docs/collaborate/explore-multiple-projects.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+
+When viewing the resource-level lineage graph for a given project that uses cross-project references, you can see cross-project relationships represented in the DAG. The iconography is slightly different depending on whether you're viewing the lineage of an upstream producer project or a downstream consumer project. 
+
+When viewing an upstream (parent) project that produces public models that are imported by downstream (child) projects, public models will have a counter icon in their upper right corner that indicates the number of projects that declare the current project as a dependency. Selecting that model reveals the lineage to show the specific projects that are dependent on this model. Projects show up in this counter if they declare the parent project as a dependency in its `dependencies.yml` regardless of whether or not there's a direct `{{ ref() }}` against the public model. Selecting a project node from a public model opens the resource-level lineage graph for that project, which is subject to your permissions.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-multiple-projects.md]:
```suggestion
When you view an upstream (parent) project, its public models display a counter icon in the upper right corner indicating how many downstream (child) projects depend on them. Selecting a model reveals the lineage indicating the projects dependent on that model. These counts include all projects listing the upstream one as a dependency in its `dependencies.yml`, even without a direct `{{ ref() }}`. Selecting a project node from a public model opens its detailed lineage graph, which is subject to your [permission](/docs/cloud/manage-access/enterprise-permissions).
```

File: website/docs/docs/collaborate/explore-multiple-projects.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="80%" height="100" title="Cross-project lineage in a parent project"/>
+
+When viewing a downstream (child) project that imports and refs public models from upstream (parent) projects, public models will show up in the lineage graph and display an icon on the graph edge that indicates what the relationship is to a model from another project. Hovering over this icon indicates the specific dbt Cloud project that produces that model. Double-clicking on a model from another project opens the resource-level lineage graph of the parent project, which is subject to your permissions.
+
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-child.png" width="85%" height="100" title="Cross-project lineage in a child project"/>
+
+## Explore the project-level lineage graph
+
+For cross-project collaboration, you can interact with the DAG in all the same ways as described in [Explore your project's lineage](#project-lineage) but you can also interact with it at the project level and view the details.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-multiple-projects.md]: this link doesnt do anything -- did you mean this link?

```suggestion
For cross-project collaboration, you can interact with the DAG in all the same ways as described in [Explore your project's lineage](/docs/collaborate/explore-projects#project-lineage) but you can also interact with it at the project level and view the details. 
```

File: website/docs/docs/collaborate/explore-multiple-projects.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-child.png" width="85%" height="100" title="Cross-project lineage in a child project"/>
+
+## Explore the project-level lineage graph
+
+For cross-project collaboration, you can interact with the DAG in all the same ways as described in [Explore your project's lineage](#project-lineage) but you can also interact with it at the project level and view the details. 
+
+To get a list view of all the projects, select the account name at the top of the **Explore** page near the navigation bar. This view includes a public model list, project list, and a search bar for project searches. You can also view the project-level lineage graph by clicking the Lineage view icon in the page's upper right corner.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-multiple-projects.md:25]: non blocker - i dont think i understand how to get to the project-level lineage graph. i don't see it on the upper right corner. is it this overview page?

Screenshot 2023-11-28 at 06 54 58

File: website/docs/docs/collaborate/explore-multiple-projects.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+
+If you have permissions for a project in the account, you can view all public models used across the entire account. However, you can only view full public model details and private models if you have permissions for a project where the models are defined.
+
+From the project-level lineage graph, you can:
+
+- Click the Lineage view icon (in the graph’s upper right corner) to view the cross-project lineage graph.
+- Click the List view icon (in the graph’s upper right corner) to view the project list.
+    - Select a project from the **Projects** tab to switch to that project’s main **Explore** page.
+    - Select a model from the **Public Models** tab to view the [model’s details page](#view-resource-details).
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-multiple-projects.md]: updating link
```suggestion
    - Select a model from the **Public Models** tab to view the [model’s details page](/docs/collaborate/explore-projects#view-resource-details).
```

**Review by mirnawong1 — APPROVED** [review]
some suggestions so content is easier to read and fixing some links up too

**Comment by mirnawong1** [comment]
hey @nghi-ly and @cafzal - committed the changes, especially the links, and will merge this now

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Enhancement: New features and UI changes to dbt Explorer"
+description: "November 2023: New features and UI changes to dbt Explorer, including a new filter panel, improved lineage graph, and detailed resource information."
+sidebar_label: "Enhancement: New features and UI changes to dbt Explorer"
+sidebar_position: 08
+tags: [Nov-2023]
+---
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md:7]: i think the date property was missed! might be good to post in the dbt cloud community channel

File: website/docs/docs/collaborate/explore-multiple-projects.md (hunk 15 above)
Comment (nghi-ly) [reply website/docs/docs/collaborate/explore-multiple-projects.md:25]: chatted with mirna about this during our 1-1. resolving.

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md (hunk 17 above)
Comment (nghi-ly) [reply website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md:7]: great flag! will update in a fast-follow pr, tysm!

**Pull request merged**